statistics.test.py

```diff
@@ -16,6 +16,10 @@ def test_avg_is_nan_for_empty_input(self):
     # nan (not-a-number), as defined in the math package
     # Specify the assert here.
     # Use nan and isnan in https://docs.python.org/3/library/math.html
+    import math
```

Comment:
Good way to keep the import local to the block where you use it. This way, if you change the approach, you naturally change the dependency as well